mint.json

```diff
@@ -514,9 +514,7 @@
         "recipes/workflows/otp",
         "recipes/workflows/password-reset",
         "recipes/workflows/recent-login",
-        "recipes/workflows/invoice-receipt",
-        "recipes/workflows/shipping-confirmation",
-        "recipes/workflows/feedback-reviews"
```

Comment:
What about those workflows? Any reason to delete them?

Reply:
By mistake